article/app/views/fragments/articleBodyGarnett.scala.html

```diff
@@ -28,7 +28,7 @@
             "content",
             "content--article",
             s"content--pillar-${article.metadata.pillar.map(_.name).getOrElse("News").toLowerCase}",
-            s"content--type-${article.metadata.contentType.map(_.name).getOrElse("Article").toLowerCase}",
+            s"content--type-${article.metadata.designType.map(_.toString).getOrElse("Article").toLowerCase}",
```

Comment:
might be a good idea top extract this logic somewhere since it seems to be used in multiple places

Reply:
why not use `nameOrDefault`?